packages/scripts/config/webpack.config.js

```diff
@@ -265,6 +265,17 @@ const config = {
 						return content;
 					},
 				},
+				{
+					from: '**/*.php',
+					context: process.env.WP_SRC_DIRECTORY,
+					noErrorOnMissing: true,
+					filter: ( filepath ) => {
+						return (
+							process.env.WP_COPY_PHP_FILES_TO_DIST ||
+							renderPaths.includes( filepath )
```

Comment:
Have you tested this without any `render.php` file? I think this will crash if there is no render file, because when that happens, you're returning undefined in the `getRenderPropPaths` function.

You could check for the existence of `renderPaths` here before using `includes` (`renderPaths && renderPaths.includes( filepath )`), but I'd rather always return an array in the `getRenderPropPaths` function.

Reply:
You're totally right. I have just changed the function with your suggestions to ensure it always returns an array, even if it's empty. I have tested it and it seems to work when there are no `render` props defined.